### ISSUE TITLE
License

### ISSUE THREAD

**Issue by owahltinez** [covid19-eu-zh/covid19-eu-data#57]
Hello, thanks for all the great work in this repo! Can you please provide an explicit license for the data? Preferably, something with as few restrictions as possible...

**Comment by emptymalei** [comment]
Good question.

The code itself shall be MIT. 

However, this repo has data in it. We are using many different sources. 
Apart from CH and IT, we get the data from gov websites. I believe these shall be attributed as open data with some kind of open license (?).  
IT data is [CC](https://github.com/pcm-dpc/COVID-19/blob/master/LICENSE). CH data source [doesn't specify](https://github.com/daenuprobst/covid19-cases-switzerland).

😕

**Comment by owahltinez** [comment]
Thanks! If I may, I would suggest to use [OpenZH](http://github.com/openZH/covid_19) as the data source for CH, which also has a very clear license.

**Comment by emptymalei** [comment]
@owahltinez Good point. I just took a loot at the [data source](https://github.com/daenuprobst/covid19-cases-switzerland/blob/master/covid_19_data_switzerland.xlsx) we are using. The data is a [transformation](https://github.com/daenuprobst/covid19-cases-switzerland/blob/0a572cf70b587a8a1a00ff864849c5f356b436a8/pull_ozh.py#L126) of openzh. Following the definition of CC BY, this data we have here should also be CC BY.

Looks like the license problem is solved. ✌️
